daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/NamedEntity.scala

```diff
@@ -29,22 +30,6 @@ object NamedEntity {
     def pretty: String = s"module $name"
   }
 
-  final case class NValDef(
```

Comment:
did we ever use this for anything?

Reply:
Yes we use to have it when we implement collision, but we quickly drop collision between data types and values as it was annoying for the compiler.